### DISCUSSION TITLE
How to set the foreground color of Uno.Toolkit.UI.NavigationBar.Subtitle in Dark theme?

### DISCUSSION THREAD

**Discussion by ForkBug** [unoplatform/uno#19095] [Q&A] [answered]
The Subtitle is invisible in the dark theme on Android when Content is visible.

**Comment by morning4coffe-dev** [comment, marked as answer]
Hey @ForkBug,

Thank you for reporting this! I am able to confirm that this issue is happening on WinUI and Skia (desktop) with Material Light/Dark theme. I will convert this into an issue in Toolkit repo to track this further. Feel free to provide any more details there.

**Reply by morning4coffe-dev** [reply]
Issue: https://github.com/unoplatform/uno.toolkit.ui/issues/1311
